src/ima.js

```diff
@@ -326,6 +348,27 @@ class Ima extends BasePlugin implements IMiddlewareProvider, IAdsControllerProvi
     return new ImaAdsController(this);
   }
 
+  /**
+   * Gets the indicator if ads still playing on the same player.
+   * @public
+   * @returns {boolean} - if ads still playing on the same player.
+   * @instance
+   * @memberof Ima
+   */
+  isAdsPlayingCustomPlayback(): boolean {
```

Comment:
custom playback is ima term and says nothing to me. change it to a meaningful name